<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>澳门项目app控制云碟调试工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>APP控制云碟测试</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>协议类型：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="m_pComboBox">
              <item>
               <property name="text">
                <string>UDP</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>TCP</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Key加密：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_pEncryptIn"/>
            </item>
            <item>
             <widget class="QPushButton" name="m_pButtonEncrypt">
              <property name="text">
               <string>加密</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_pEncryptOut"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Key解密：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_pDecryptIn"/>
            </item>
            <item>
             <widget class="QPushButton" name="m_pButtonDecrypt">
              <property name="text">
               <string>解密</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_pDecryptOut"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>发送消息</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>服务器IP:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_pServerIP"/>
            </item>
            <item>
             <widget class="QPushButton" name="m_pButtonSend">
              <property name="text">
               <string>点击发送</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLineEdit" name="m_pTextSend"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4"/>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>接收消息</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="m_pTextReceive"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>ARP</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>IP网段：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_pLineIP">
            <property name="text">
             <string>192.168.1.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_pButtonArpSend">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>过滤：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_pEditFilter"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="m_pTableIP">
        <property name="minimumSize">
         <size>
          <width>589</width>
          <height>0</height>
         </size>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>160</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>140</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>30</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>100</number>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">IP</string>
         </property>
         <property name="toolTip">
          <string/>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">MAC</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
